code/game/objects/item.dm

```diff
@@ -475,19 +475,12 @@
 	if(!inv_slot)
 		return FALSE
 
-	var/already_equipped = inv_slot.get_equipped_item()
-	if(!ignore_equipped || already_equipped != src)
-		if(already_equipped)
-			if(!force)
-				return FALSE
-			inv_slot.clear_slot()
-			qdel(already_equipped)
-
-		if(!inv_slot.is_accessible(M, src, disable_warning))
+	if(!force)
+		if(!ignore_equipped && !inv_slot.is_accessible(M, src, disable_warning))
 			return FALSE
 
-	if(!inv_slot.can_equip_to_slot(M, src, disable_warning, ignore_equipped))
-		return FALSE
+		if(!inv_slot.can_equip_to_slot(M, src, disable_warning, ignore_equipped))
```

Comment:
Worth noting that ignoring the ``can_equip_to_slot`` check if ``force`` is true is a change in behavior. It'll allow you to do things like equip belts and IDs without a jumpsuit, rather than just ignoring if an item is already equipped in the slot.

Reply:
I feel like things that want to ignore the equipped item/covering check but not the can_equip_to_slot check shouldn't be using force, honestly.